css/grayscale.scss

```diff
@@ -360,10 +282,6 @@ footer {
     background: rgba($light, 0.2);
 }
 
-img {
```

Comment:
We need that, in order to make the images in the posts responsive. It's a fix for issue #36